### DISCUSSION TITLE
Compatibility Test

### DISCUSSION THREAD

**Discussion by wells5150** [sn4k3/UVtools#642] [Q&A] [answered]
I ran the compatibility test on on my Phrozen Mighty 8K. It goes up to 4 sec, then displays 'error'.  The head remains in the lowered position until I reboot the printer. Is this normal?
Head lowers. Light comes on for 5 sec, 2, then 4 seconds & ERROR.

**Comment by sn4k3** [comment, marked as answer]
New firmwares prevent such test and make some UVtools features not possible, like multiple  exposure tests

**Reply by wells5150** [reply]
Thanks. I will contact Phrozen.

**Reply by sn4k3** [reply]
> I will contact Phrozen.

Will make no difference, someone already did and they don't have control over firmware.

![image](https://user-images.githubusercontent.com/113281/211216148-25bfe4a5-1401-4de9-b17c-a95b2de20ccd.png)

**Reply by wells5150** [reply]
Alrighty then. My work is done. I will keep an eye out for updates.
Thanks!

**Reply by AnMontes** [reply]
What firmware version are you using in your Mighty 8K? I'm running V136b and looks to be working.

**Comment by wells5150** [comment]
Same version. It's kinda wonky. A lot of bugs, causing other errors. A lot of threads have made Phrozen aware. I will attempt again near future. Should I run the exposure test without the compatibility test?
